File: pertemuan4/harga-barang-while/harga_while.c
#include <stdio.h>

int main() {
    float harga, total = 0;
    char lanjut;

    do {
        printf("Masukkan harga barang (atau -1 untuk selesai): ");
        scanf("%f", &harga);

        if (harga != -1) {
            total += harga;
        }
    } while (harga != -1);

    printf("Total harga: %.2f\n", total);
    return 0;
}

File: pertemuan4/harga-barang-while/harga_while.js
let total = 0;
let input;

while (true) {
  input = prompt("Masukkan harga barang (atau 'selesai' untuk berhenti): ");
  if (input.toLowerCase() === "selesai") break;
  total += parseFloat(input);
}

console.log(`Total harga: ${total.toFixed(2)}`);
